src/script/interpreter.cpp

```diff
@@ -1354,11 +1354,15 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                     popstack(stack);
                     popstack(stack);
                     stack.push_back(fSuccess ? vchTrue : vchFalse);
+
                     if (opcode == OP_CHECKSIGFROMSTACKVERIFY)
-                        popstack(stack);
+                    {
```

Comment:
EDIT: in fact, more style nits:
```

if (opcode == OP_CHECKSIGFROMSTACKVERIFY) {
    if (fSuccess) {
        popstack(stack);
    } else {
        return set_error(serror, SCRIPT_ERR_CHECKSIGFROMSTACKVERIFY);
    }
}
```
But perhaps it is better to have it like the rest (checksigverify and OP_CHECKMULTISIGVERIFY) as an exception to the style rules?